src/index.html

```diff
@@ -6,9 +6,64 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <a href="#">
+        <img
+          class="moyo-logo"
+          src="images\logo.png"
+          alt="moyo-img"
+        >
+      </a>
+
+      <nav>
+        <ul class="nav__list">
+
+          <li class="nav__item">
```

Comment:
Add blank lines between child elements but don't add between parent and child. Fix in all cases